crates/rslint_cli/src/lib.rs

```diff
@@ -153,6 +154,72 @@ pub fn apply_fixes(results: &mut Vec<LintResult>, walker: &mut FileWalker, dirty
     fix_count
 }
 
+pub fn dump_ast(glob: String) {
+    for_each_file(glob, |walker, file| {
+        let header = if let Some(path) = &file.path {
+            format!("File {}", path.display())
+        } else {
+            format!("File {}", file.name)
+        };
+        println!("{}", header.red().bold());
+
+        let res = if file.kind == JsFileKind::Module {
+            rslint_parser::parse_module(&file.source, file.id).to_syntax()
+        } else {
+            rslint_parser::parse_text(&file.source, file.id).to_syntax()
+        };
```

Comment:
```suggestion
        let res = file.parse();
```
I added a utility function for this in a recent commit